pkg/proxy/ipvs/proxier.go

```diff
@@ -1735,39 +1729,6 @@ func (proxier *Proxier) cleanLegacyService(atciveServices map[string]bool, curre
 	}
 }
 
-// linkKubeServiceChain will Create chain KUBE-SERVICES and link the chin in PREROUTING and OUTPUT
```

Comment:
keep the comment?

Reply:
I abstracted this part function to other place, now added comments to be more specify. : )

Reply:
We should keep this function?